stable/stolon/templates/proxy-deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
 {{ toYaml . | indent 8 }}
 {{- end }}
     spec:
+      serviceAccountName: {{ template "stolon.serviceAccountName" . }}
```

Comment:
Should you set the value of serviceAccountName to default in values.yaml? - https://github.com/kubernetes/charts/blob/master/stable/stolon/values.yaml#L33

Reply:
why? the name will be generated based on fullname.
I think behaviour is quite clear from the comment in values.yaml:
```
# The name of the ServiceAccount to use. If not set and create is true, a name is generated using the fullname template
```

Reply:
My apologies. This was misread on my side.